src/Components/Header/index.js

```diff
@@ -11,7 +11,7 @@ const Header = ({openComponent}) => {
             <Button 
                 h="35px"
                 w="100px"
-                name={"+"} 
+                name={"Add new contact"} 
```

Comment:
I saw that in the top for w and h props you used propName="propValue" syntax, if you are using some syntax please keep it everywhere. Here you wrote propName={PropValue}, but the prop value is a dummy string